sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1724,12 +1724,18 @@ pi_result ExecCGCommand::SetKernelParamsAndLaunch(
 void DispatchNativeKernel(void *Blob) {
   // First value is a pointer to Corresponding CGExecKernel object.
   CGExecKernel *HostTask = *(CGExecKernel **)Blob;
+  bool MayDeleteCG = static_cast<bool>(*((void **)Blob + 1));
```

Comment:
Obfuscation style! :-)
```suggestion
  bool MayDeleteCG = *(static_cast<void **>(Blob) + 1) != nullptr;
```
or even better
```suggestion
  bool MayDeleteCG = static_cast<void **>(Blob)[1] != nullptr;
```

Reply:
Thanks. Fixed.